dev/VSIX/extension.manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vsix-publish",
+    "categories": [ "Build", "Coding", "Programming Languages" ],
+    "identity": {
+        "internalName": "[Replace]"
```

Comment:
Remember to update this line before finishing.

Reply:
This was going to be updated by the pipeline. I was planning on using this manifest json for all the VSIXs that we have.